packages/dm-core/src/components/Pickers/JobHandlerPicker.tsx

```diff
@@ -26,7 +27,7 @@ export const JobHandlerPicker = (props: {
         absoluteId: `WorkflowDS/${blueprintId}`,
       })
       .then((response: any) => {
-        onChange(response.data)
+        onChange(`${protocolPrefix}${response.data}`)
```

Comment:
Should we not rather fix DMSS to return a path with the correct protocol prefix?

Reply:
mm, yeah. That makes sense.